.gitignore

```diff
@@ -38,3 +38,17 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+/.vs/slnx.sqlite
```

Comment:
Volgens mij hoef je dit soort zaken niet op te nemen in gitignore file, weet je zeker dat ze in de solution file moete zijn bij jou op lokaal.

Reply:
al verwijder ik die files, zodra ik visual studio opstart genereert hij opnieuw die bestanden, heb jij hier zo een oplossing voor? Ben ondertussen al zoekende.